LoveMachine.Core/ButtplugController.cs

```diff
@@ -192,6 +220,23 @@ protected void DoVibrate(float intensity, int girlIndex, Bone bone)
             client.VibrateCmd(intensity, girlIndex, bone);
         }
 
+        protected internal IEnumerator DoRotate(int girlIndex, Bone bone,
+                    float strokeTimeSecs)
+        {
+
+            float speed = Mathf.Lerp(0.3f, 1f, 0.4f / strokeTimeSecs) * CoreConfig.RotationSpeedRatio.Value;
+            client.RotateCmd(speed: speed, clockwise: clockwise, girlIndex, bone);

```

Comment:
I normally try to avoid using named arguments when they already have the same name as the parameter because it feels redundant.